modules/chat/public/views/pages/inbox.html.liquid

```diff
@@ -0,0 +1,77 @@
+---
+slug: inbox
```

Comment:
I don't feel that we should have pages in modules, even partials with html files.